docs/core-concepts/core-sdk/cli/extensibility.md

```diff
@@ -0,0 +1,174 @@
+# .NET Core CLI extensibility model 
+
+## Overview
+This document will cover the main ways how to extend the CLI tools. It will cover the two main ways on how to extend 
```

Comment:
Seems repetitive. Maybe something like "This document will cover the two main ways on how to extend the CLI tools and explain the scenarios that drive each of them. "